### DISCUSSION TITLE
2023 October Office Hours

### DISCUSSION THREAD

**Discussion by scottyhq** [pydata/xarray#8220] [Office Hours]
Opening a thread here for planning more virtual office hours over the coming months. These events are designed to create welcoming spaces for users around the world to collaborate and gain experience with Xarray! 

I'm hoping to get something on the calendar for mid-October, and would greatly appreciate anyone interested in helping out to give a +1 on this post. 

**October 11, 9-11am PDT Register Here!**
https://www.eventbrite.com/e/xarray-virtual-office-hours-october-2023-tickets-728953317997

~~Also select your preferred times here (http://whenisgood.net/xarray-office-hours) if you're interested in attending. Early next week I'll post a registration link for a 2 hour block of time.~~

As in the past, we'll request attendees identify issues or discussion threads to work on in advance so that we can ensure enough hands are on deck to break into small groups to work together. Thanks!

**Comment by douglatornell** [comment]
+1

I'd like to try to resolved the time offsetting issue in resampling that arises from the pending removal of the `loffset` parameter in `Dataset.resample()`. I've described the issue in discussion thread #8175. Perhaps, if I can get that sorted out I can try to contribute to documentation for time offsets (issue #7596).

**Reply by scottyhq** [reply]
Pinging some past organizers to help settle on a time for this (survey in top comment). @dcherian @JessicaS11 @andersy005 @keewis @TomNicholas @mgrover1 @betolink @maxrjones hope you can join for another round!

**Reply by keewis** [reply]
I'd most likely be available (especially before Oct 15th), but it depends on when exactly.

**Reply by kmuehlbauer** [reply]
I'm available after October 15th but it also depends on the exact times (located in Europe).

**Reply by scottyhq** [reply]
Always hard to find a time that works for everyone, but based on the scheduling poll for this round we'll go with Wednesday October 11, 9-11am PDT. I'll post an eventbrite registration link in the top comment!

**Reply by mgrover1** [reply]
I am available to help @scottyhq !

**Reply by dcherian** [reply]
Unfortunately I'll be out of town :(. Good luck!

**Comment by JessicaS11** [comment]
One topic that came up during office hours: needing better cross-linking of docs and tutorial material (for `apply_ufunc`), i.e. #8008.

**Reply by Evidlo** [reply]
Did office hours end early?  I thought they were 11am-1pm Central?

**Reply by JessicaS11** [reply]
You're correct - we just wrapped up (so ended ~10 minutes early). Did you have issues joining?
